pkg/controller/replication_controller.go

```diff
@@ -366,6 +376,13 @@ func (rm *ReplicationManager) syncReplicationController(key string) error {
 		return err
 	}
 	controller := *obj.(*api.ReplicationController)
+	if !rm.podStoreSynced() {
+		// Sleep so we give the pod reflector goroutine a chance to run.
+		time.Sleep(PodStoreSyncedPollPeriod)
+		glog.Infof("Waiting for pods controller to sync, requeuing rc %v", controller.Name)
```

Comment:
I figured we shouldn't see more than a few because of the time.Sleep. If I upload another pr it's going to go through another set of testing, which might need more waiting/button pushing before merge :\